releasenotes/notes/bump-horizon-cloudkitty-dashboards-e94f706d5a993cc9.yaml

```diff
@@ -0,0 +1,5 @@
+---
+upgrade:
+  - |
+    Brings in commit 8c2aa37 for cloudkitty-dashboard, which includes the
```

Comment:
Rather than mention a commit SHA, you could instead link to the upstream change on Gerrit.

Reply:
Done